tests/int/utils/horti.js

```diff
@@ -54,5 +58,8 @@ module.exports = {
         }
       });
     });
+  },
+  cleanWorkingDir: () => {
+    return new Promise((yay, nay) => fs.remove('./test-workspace', err => err ? nay(err) : yay()));
```

Comment:
Consider using utils.promisify here - I think it's slightly cleaner and less error prone, eg:

```js
  const {promisify} = require('util');
  //...
  cleanWorkingDir: () => promisify(fs.remove)('./test-workspace')
```

Reply:
oh nice, did not know about this